client/extensions/woocommerce/woocommerce-services/state/shipping-label/actions.js

```diff
@@ -788,35 +788,42 @@ const handlePrintFinished = ( orderId, siteId, dispatch, getState, hasError, lab
 	if ( shouldEmailDetails( getState(), orderId, siteId ) ) {
 		const trackingNumbers = labels.map( label => label.tracking );
 		const carrierId = first( labels ).carrier_id;
```

Comment:
Might need to figure out why we use `first(labels)`

Reply:
Hmm, yeah, it's not handling this well:
<img width="380" alt="image" src="https://user-images.githubusercontent.com/6209518/88956415-2948a780-d252-11ea-93d2-89285c577ae2.png">

Reply:
This will be pretty unusual, so I'm comfortable releasing with this bug. I'll make an issue and we can prioritize it next week

Reply:
I wonder if labels are sorted in descending order based on last process/most updated datetime.

Reply:
Issue created: https://github.com/Automattic/woocommerce-services/issues/2113

Reply:
You can have multiple labels in an order, it assumes the carrier of the first label is the carrier for all labels.